Editor/Bootstrap/BootstrapSettings.cs

```diff
@@ -214,9 +214,11 @@ private void SetErrorStatus(string statusKey)
 
         private void SetErrorStatus(string statusKey, Response errorResponse)
         {
+            errorResponse.ErrorMessage = _textProvider.GetError(errorResponse.ErrorCode);
             string errorTemplate = _textProvider.Get(statusKey);
-            string message = string.Format(errorTemplate, _textProvider.GetError(errorResponse.ErrorCode),
+            string message = string.Format(errorTemplate, errorResponse.ErrorCode,
                 errorResponse.ErrorMessage);
+            
             _status.SetMessage(message, MessageType.Error);
             _status.IsDisplayed = true;
         }
```

Comment:
What is the purpose of Status compared to StatusBox? I see there's a related method in status box that handles the message formatting (additionalText): https://github.com/aws/amazon-gamelift-plugin-unity/blob/9a46258d8c75ce7d5022bb37dd75e237fb811e9d/Editor/StatusBox.cs#L65

Reply:
Status is legacy code. StatusBox is the new plugins version

Reply:
Got it, could you provide a screenshot comparing this to the StatusBox? I want to confirm that the UX looks the same - in which case this would be fine. If not, we'd need a follow up task on our end to make both consistent.

Reply:
We are closing this out for now. Creating a task on our end to figure out the right way to deal with error messages thrown by the plugin window after consulting with UX.